site/_docs/configuration.md

```diff
@@ -214,7 +214,7 @@ class="flag">flags</code> (specified on the command-line) that control them.
     <tr class="setting">
       <td>
         <p class="name"><strong>Future</strong></p>
-        <p class="description">Publish posts with a future date.</p>
+        <p class="description">Publish posts or any collection with a future date.</p>
```

Comment:
@DirtyF Not at all! It was fine English as it was, but I nitpick the docs to see if we can be more precise. Thanks for submitting this patch! :heart: